website/docs/community/bindings.md

```diff
@@ -80,6 +80,7 @@ As a C library, bindings can be made to call H3 functions from different program
 
 - [uber/h3-py](https://github.com/uber/h3-py)
 - [DahnJ/H3-Pandas](https://github.com/DahnJ/H3-Pandas)
+- [h3-py on Conda Forge](https://anaconda.org/conda-forge/h3-py) ([conda-forge/h3-py-feedstock](https://github.com/conda-forge/h3-py-feedstock))
```

Comment:
We probably want to also sync this with the 3.x docs? There's some confusion about bindings not showing up in different API versions https://github.com/uber/h3/commit/14865690c8f68f39c4d16d1019d6b4817c77e90f (perhaps we just symlink the `bindings.md` between the two directories, or maybe even the entire `community` directory?

Reply:
I wonder if it's possible to take this section out of the "versioned" docs? Only the API really needs to be versioned.